node_status_prom_file.py

```diff
@@ -0,0 +1,40 @@
+import subprocess
+import json
+from datetime import datetime
+
+metrics_file_name = "node-status-file.prom"
+command = ""
+
+check_status = subprocess.run(
+    [command, "status"], capture_output=True, text=True)
+
+if check_status.returncode == 0:
+    if check_status.stdout:
+        print("stdout")
+        parse_check_status = json.loads(check_status.stdout)
+    elif check_status.stderr:
+        print("stderr")
+        parse_check_status = json.loads(check_status.stderr)
+else:
+    exit()
```

Comment:
Is this an error exit? If so return code should not be non-zero and should give some form of explanation as to why the process exited (or even just printing the errors out).